peer/brontide.go

```diff
@@ -2437,19 +2438,14 @@ out:
 			// values, as they currently are always set to the default values
 			// at initial channel creation. Note that the maximum HTLC value
 			// defaults to the cap on the total value of outstanding HTLCs.
-			//
-			// TODO(guggero): We should instead pass in the current
-			// forwarding policy from the funding manager to avoid
-			// needing us to update the link once the channel is
-			// announced to the network with custom user values.
-			fwdMinHtlc := lnChan.FwdMinHtlc()
-			defaultPolicy := p.cfg.RoutingPolicy
```

Comment:
the removal of `RoutingPolicy` made me think should we remove it? It seems that it's only used when we load channels from startup, and we should instead read the routing policy from db? Needs more check here, looks like the `RoutingPolicy` is created in `server.go` when reading bitcoin configs...

Reply:
That's a good idea, though it looks like it's still used in `loadActiveChannels`.

Reply:
Took a look at this again. We are reading the routing policies from the DB, but unfortunately they are only stored in the graph DB and not in the actual channel DB. And it can happen that there is no outgoing channel policy for some reason (I've seen some issues around that), so being able to fall back to the default policy instead of failing the channel sounds like a good reason to keep it.